demo/deployment.yaml

```
@@ -0,0 +1,67 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: podinfo
+  namespace: demo
+  labels:
+    app: podinfo
+spec:
+  minReadySeconds: 5
+  revisionHistoryLimit: 5
+  progressDeadlineSeconds: 60
+  strategy:
+    rollingUpdate:
+      maxUnavailable: 0
+    type: RollingUpdate
+  selector:
+    matchLabels:
+      app: podinfo
+  template:
+    metadata:
+      annotations:
+        prometheus.io/scrape: "true"
+      labels:
+        app: podinfo
+    spec:
+      containers:
+      - name: podinfod
+        image: quay.io/stefanprodan/podinfo:2.1.0
```

```suggestion
        image: stefanprodan/podinfo:2.1.0
```
Better use Docker Hub, so that Flux will not be rate limited by Quay.

---

Good point. I copied these manifests from the podinfo repo, maybe we should also update the image there.

---

I've changed in podinfo repo.